### DISCUSSION TITLE
hide ports from output

### DISCUSSION THREAD

**Discussion by gprime31** [projectdiscovery/tlsx#300] [Ideas]
current:
https://site.com:443
https://site1.com:443
https://site2.com:443

would like:
https://site.com
https://site1.com
https://site2.com

or maybe i missed the option to do this already?

**Comment by ehsandeep** [comment]
@gprime31 no, it has no option to hide it as it's crucial information to display to confirm which port was used to perform tls scanning.

Can you let us know the use case or issue you are facing by having port information?

**Reply by gprime31** [reply]
Because when I anew it to a subdomain list, now I have to remove duplicates.
Example: 
https://site.com
https://site.com:443
It would just save a step.
Also I've used tlsx with -p {port,port,port} for a while and have not seen anything other than port 443 show up.
Thanks.

**Reply by ehsandeep** [reply]
tlsx is / should not be used for subdomain enum or the collection, default output for tlsx is formatted for what is should be used for, i.e scanning for tls certificates for the given `host:port` pair, I'm not sure what's your workflow look like, but this not the intended use of tlsx.

as an alternative, you can use `-j` option to output in any format you want.

```
echo hackerone.com | tlsx -j | jq -r .host
```

**Comment by gprime31** [comment]
really? I use it like:

echo hackerone.com | tlsx -port 8080,8081,8443,9443,80,443

but your alternative will work,

thanks.
